### PR TITLE
Bugfix/fix issue 1

### LINKED ISSUE #1: Error on NixOS 22.05
While trying to run the command
```
nix run github:gvolpe/falsisign.nix#falsisign -- \   
    -d document.pdf \
    -s Signature_example \
    -x 1000 -y 2500 \
    -p '1' \
    -o scanned.pdf
```
I got
```
error: attribute 'apps.x86_64-linux.falsisign' should have type 'app'
```

If you need further information please feel free to ask. i am not very savvy with flakes yet so I am not really able to provide a PR.

### LINKED ISSUE #3: Error with Nix 2.8
Running
```
$ > nix run flake.nix#signdiv /tmp/falsisign/Signature_guide.pdf 
```
yields
```
path '/tmp/falsisign.nix/flake.nix' does not contain a 'flake.nix', searching up
error: attribute 'apps.x86_64-linux.signdiv' should have type 'app'
```

### PR DESCRIPTION
**Pull request by madonius**
Closes #1 
Closes #3

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -23,13 +23,16 @@
             inherit pkgs falsisign-src;
           };
         in
-        {
-          apps = {
+		{
+		  apps.x86_64-linux= {
```
Comment (gvolpe) [inline flake.nix]: This should probably be `apps.${system} = { ... } `, otherwise this will only work on this specific platform.

**Review by gvolpe — CHANGES_REQUESTED** [review]

File: flake.nix (hunk 1 above)
Comment (madonius) [reply flake.nix]: Fixed

**Comment by madonius** [comment]
Closes #3 too

**Review by gvolpe — APPROVED** [review]
Thank you!

**Pull request merged**